ci/builders/mac_host_engine.json

```diff
@@ -12,56 +75,562 @@
+                "$flutter/osx_sdk": {
+                    "sdk_version": "15a240d"
+                }
+            },
+            "timeout": 90
+        },
+        {
+            "archives": [
+                {
+                    "base_path": "out/ci/host_release/zip_archives/",
+                    "type": "gcs",
+                    "include_paths": [
+                        "out/ci/host_release/zip_archives/darwin-x64-release/artifacts.zip",
+                        "out/ci/host_release/zip_archives/darwin-x64/font-subset.zip"
+                    ],
+                    "name": "ci/host_release",
+                    "realm": "production"
+                }
+            ],
+            "drone_dimensions": [
+                "device_type=none",
+                "os=Mac-13|Mac-14"
+            ],
+            "dependencies": [
+                {
+                    "dependency": "goldctl",
+                    "version": "git_revision:720a542f6fe4f92922c3b8f0fdcc4d2ac6bb83cd"
+                }
+            ],
+            "gclient_variables": {
+                "download_android_deps": false,
+                "use_rbe": true
+            },
+            "gn": [
+                "--target-dir",
+                "ci/host_release",
+                "--runtime-mode",
+                "release",
+                "--no-lto",
+                "--prebuilt-dart-sdk",
+                "--build-embedder-examples",
+                "--use-glfw-swiftshader",
+                "--rbe",
+                "--no-goma",
+                "--xcode-symlinks"
+            ],
+            "name": "ci/host_release",
+            "description": "Produces release mode x64 macOS host-side tooling.",
+            "ninja": {
+                "config": "ci/host_release",
+                "targets": [
+                    "flutter/build/archives:artifacts",
+                    "flutter/tools/font_subset"
+                ]
+            },
+            "postsubmit_overrides": {
+                "gn": [
+                    "--target-dir",
+                    "ci/host_release",
+                    "--runtime-mode",
+                    "release",
+                    "--no-lto",
+                    "--prebuilt-dart-sdk",
+                    "--build-embedder-examples",
+                    "--use-glfw-swiftshader",
+                    "--no-rbe",
+                    "--no-goma"
+                ]
+            },
+            "properties": {
+                "$flutter/osx_sdk": {
+                    "sdk_version": "15a240d"
+                }
+            },
+            "timeout": 90
+        },
+        {
+            "drone_dimensions": [
+                "device_type=none",
+                "os=Mac-13|Mac-14"
+            ],
+            "dependencies": [
+                {
+                    "dependency": "goldctl",
+                    "version": "git_revision:720a542f6fe4f92922c3b8f0fdcc4d2ac6bb83cd"
+                }
+            ],
```

Comment:
I don't think this one tests goldens, but I may be missing it:
https://ci.chromium.org/ui/p/flutter/builders/try/Mac%20Engine%20Drone/1019975/overview